it/test/uk/gov/hmrc/tradergoodsprofilesrouter/CreateRecordIntegrationSpec.scala

```diff
@@ -928,6 +928,48 @@ class CreateRecordIntegrationSpec
         |}
         |""".stripMargin)
 
+  lazy val createRecordResponseData: JsValue =
```

Comment:
this does not need to be lazy. In know this was already like that before. Would you mind to make the changes please?